interfaces/builtin/unity8.go

```diff
@@ -0,0 +1,204 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016-2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/snapcore/snapd/interfaces"
+)
+
+var unity8ConnectedPlugAppArmor = []byte(`
+# Description: Can access unity8 desktop services
+# Usage: common
+
+  #include <abstractions/dbus-session-strict>
+  #include <abstractions/fonts>
+
+  # qmlscene (a common client) wants this
+  network netlink raw,
```

Comment:
Ah, I figured out why this was needed.  Turns out it isn't, and I will remove.  But backstory:

I tested first with unity8+x11 before moving to unity8+mir.  Turns out that under X11, QInputInfo (in qtsystems) will call udev_monitor_enable_receiving, which connects to udev via a netlink socket.  Under Mir, it doesn't do this.  So this isn't a unity8 interface problem, and I'll drop this from here.

*However*, maybe this bit should be added to the x11 and/or unity7 interfaces.  Without it (or, more likely, without just adding the network interface), qml snaps can't launch.  But that's a separate PR.